planet-kaggle-blog.md

```diff
@@ -0,0 +1,69 @@
+## The Planet Kaggle
+
+![](imgs/banner.jpg)
+
+### Amazon Basin Data
+
+![](imgs/chipdesc.jpg)
+<p align="center">source: <a href="https://www.kaggle.com/c/planet-understanding-the-amazon-from-space/data">Planet</a></p>
+
+This past summer, Planet launched the *Understanding the Amazon from Space* Kaggle competition. We participated in this competition using Raster Vision, a system for analyzing aerial and satellite imagery using deep learning. Raster Vision works across several different tasks, including semantic segmentation, object detection and scene tagging as well as a range of datasets. The varied data within the Amazon competition gave us a challenging opportunity to make a robust scene tagging functionality.
+
+Planet provided over 100,000 chips from large images taken by a flock of satellites over the Amazon basin in 2016. These 40,000 training and 60,000 testing chips were given in both 3-band RGB JPEG and four band IR-RBG TIFF formats. The goal of tagging is to infer a set of labels for a given chip. For the Amazon images, there were 17 possible tags, which could be broadly split into three categories.
+
+1. **atmospheric labels**: clear, partly cloudy, cloudy and hazy
+2. **common labels**: primary, water, habitation, agriculture, road, cultivation and bare ground
+3. **rare labels**: artisinal mine, blooming, blow down, conventional mine, selective logging and slash burn.
+
+Each of the chips were labeled with ground truth tags through crowd-sourced labor. The ground truth is used to train models on the correct labels for a given chip. A well trained network can accurately predict the labels on a previously unseen chip.
+
+<p align="center"><img src="imgs/tags_hist.png" alt="Tag histogram"/></p>
```

Comment:
Typically histograms like this are sorted by frequency (most frequent bars to least) which makes it easier to parse. I remember seeing the same graph, except sorted on Planet's Jupyter notebook.